upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -937,12 +937,6 @@ func setupControlPlane(opt *NewClusterOptions, cluster *api.Cluster, zoneToSubne
 				g.Spec.Zones = []string{zone}
 			}
 
-			if cluster.IsKubernetesLT("1.27") && cloudProvider == api.CloudProviderAWS {
-				g.Spec.InstanceMetadata = &api.InstanceMetadataOptions{
-					HTTPTokens: fi.PtrTo("required"),
-				}
-			}
```

Comment:
I'm wondering if this should have been IsKubernetesGTE ? in which case we'd want to unconditionally set it. or rather, reintroduce it for k8s 1.34 and later unconditionally set it once we remove support for k8s 1.34. Same with the other InstanceMetadata assignments below.

Reply:
@rifelpet it is already the default, putting it in the spec makes it very hard to change it if needed for addons:
https://github.com/kubernetes/kops/blob/4d977166f31cc349b985f1885e80ecc0d14d8736/pkg/model/awsmodel/autoscalinggroup.go#L192-L193